airbase/pom.xml

```diff
@@ -148,7 +148,7 @@
         <air.check.fail-checkstyle>${air.check.fail-basic}</air.check.fail-checkstyle>
 
         <!-- minimum maven version to build -->
-        <air.maven.version>3.6.3</air.maven.version>
+        <air.maven.version>3.9.8</air.maven.version>
```

Comment:
How old is 3.9.8?

Reply:
It's not about how old it is. I'd like to see mvnd supported across the board and 1.0.0 (reporting as 3.9.8) is the first version I actually trust to deliver a value

Reply:
Age matters. if 3.9.8 is relatively recent and for example we uncover some edge case bug in 3.9.8 and there is no other version we can use, we actively prevented ourselves from using an older version as a workaround

Reply:
This can be overridden if needed with the property.